testing/trino-testing/src/main/java/io/trino/testing/DistributedQueryRunner.java

```diff
@@ -97,6 +100,18 @@
         return new Builder<>(defaultSession);
     }
 
+    public static Logging setupLogging()
+    {
+        Logging logging = Logging.initialize();
+        logging.setLevel("Bootstrap", WARN);
+        logging.setLevel("io.airlift.bootstrap", WARN);
+        logging.setLevel("org.glassfish.jersey.internal.inject.Providers", ERROR);
+        logging.setLevel("org.glassfish.jersey.internal.Errors", ERROR);
+        logging.setLevel("org.eclipse.jetty.server", WARN);
+        logging.setLevel("io.trino.server.PluginManager", WARN);
+        return logging;
```

Comment:
No need to return this. `Logging.initialize()` returns the singleton. If anyone needs it they can just ask for it.